### PR TITLE
Add wallet UI components

### PR DESCRIPTION
**Pull request by pgarciaegido**
Moving Wallet UI components to react package and common utils to the common package

Tested in starters. It working alright 

Screenshot 2023-03-14 at 13 40 22

### CODE REVIEW

File: packages/ui/react-ui/src/CrossmintNFTCollectionView.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    assertValidNFTCollectionViewProps(props);
+
+    const src = getNFTCollectionViewSrc(props, LIB_VERSION);
+
+    return (
+        <iframe
+            src={src}
+            width="100%"
+            height="100%"
+            style={{
```
Comment (mPaella) [inline packages/ui/react-ui/src/CrossmintNFTCollectionView.tsx:21]: its not `display: flex` so flex grow doesnt do anything i think

File: packages/ui/react-ui/src/CrossmintNFTCollectionView.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                flexGrow: "1",
+                border: "none",
+                margin: "0",
+                padding: "0",
+            }}
+            role="nft-collection-view"
+        ></iframe>
```
Comment (mPaella) [inline packages/ui/react-ui/src/CrossmintNFTCollectionView.tsx:28]: super nit: 
```suggestion
        />
```

File: packages/ui/react-ui/src/CrossmintNFTDetail.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            width="100%"
+            height="100%"
+            style={{
+                flexGrow: "1",
+                border: "none",
+                margin: "0",
+                padding: "0",
+            }}
+            role="nft-details"
+        ></iframe>
```
Comment (mPaella) [inline packages/ui/react-ui/src/CrossmintNFTDetail.tsx:24]: super nit:
```suggestion
        />
```

**Review by mPaella — APPROVED** [review]

File: packages/ui/react-ui/src/CrossmintNFTCollectionView.tsx (hunk 1 above)
Comment (pgarciaegido) [reply packages/ui/react-ui/src/CrossmintNFTCollectionView.tsx:21]: Actually, the display flex would be on the parent

**Pull request merged**